blockapi/v2/base.py

```diff
@@ -29,13 +29,18 @@ def get(
         self,
         request_method: str,
         headers=None,
+        params=None,
         **req_args,
     ) -> Dict:
         """
         Call specific request method with params and return raw response.
         """
         url = self._build_request_url(request_method, **req_args)
-        response = self._session.get(url, headers=headers)
+        if params:
+            response = self._session.get(url, headers=headers, params=params)
+        else:
+            response = self._session.get(url, headers=headers)
+
```

Comment:
why? `params` is by default `None`, and it's optional arg in `get` method:

```suggestion
        response = self._session.get(url, headers=headers, params=params)
```